src/Mod/Sketcher/Gui/TaskSketcherMessages.cpp

```diff
@@ -105,6 +105,25 @@ void TaskSketcherMessages::slotSetUp(const QString& state, const QString& msg, c
     ui->labelConstrainStatus->setText(msg);
     ui->labelConstrainStatusLink->setUrl(link);
     ui->labelConstrainStatusLink->setText(linkText);
+    updateToolTip(link);
+}
+
+void TaskSketcherMessages::updateToolTip(const QString& link)
+{
+    if( link == QString::fromLatin1("#conflicting"))
+        ui->labelConstrainStatusLink->setToolTip(tr("Click to select the conflicting constraints."));
+    else
+    if( link == QString::fromLatin1("#redundant"))
+        ui->labelConstrainStatusLink->setToolTip(tr("Click to select the redundant constraints."));
+    else
+    if( link == QString::fromLatin1("#dofs"))
+        ui->labelConstrainStatusLink->setToolTip(tr("The sketch has unconstrained elements giving raise to those Degrees Of Freedom. Click to select the unconstrained elements."));

```

Comment:
@abdullahtahiriyo perhaps you meant 'giving rise' instead of 'giving raise' ?

Reply:
Indeed. My bad English. I will fix it right away.

Reply:
https://github.com/FreeCAD/FreeCAD/pull/9353